client/checkout/utils/copy-test-number.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+document.addEventListener(
+	'click',
```

Comment:
Not a huge deal, but this listener fires whenever the user clicks _anywhere_ on the page. Was this a workaround for the blocks checkout, where the element doesn't render on page load and re-renders when selecting other payment gateways? I can't see an obvious and trivial to attach the listener to this specific element in the blocks checkout--and FWIW, I couldn't get the clipboard copy to trigger unexpectedly in any scenario--but just want to confirm whether my understanding of this decision is somewhat accurate. 🤔

Reply:
> this listener fires whenever the user clicks anywhere on the page

Yep, that's correct

> Was this a workaround for the blocks checkout, where the element doesn't render on page load and re-renders when selecting other payment gateways? 

Both yes and no 😅 I'm using event delegation in this case because indeed it's not trivial to add a click event listener on an element that isn't immediately rendered. But I'm also using it because it's slightly better when the element is added and removed from the page multiple times (think of switching between payment methods on blocks checkout, when the element is mounted/unmounted multiple times).
It's easy to forget to remove an event listener from the element when the parent component is unmounted, so in this case event delegation can be slightly more performant.

This is the jQuery equivalent:

```js
jQuery( document ).on( 'click', '.js-woopayments-copy-test-number', ( event ) => {...} );
```

jQuery is a little prettier than the vanilla JS one, but they do the same thing.